src/swagger/user.doc.js

```diff
@@ -79,6 +79,136 @@ const paths = {
       },
     },
   },
+  '/api/users/profile': {
+    get: {
+      tags: ['Users'],
+      summary: 'Retrieve all profiles',
+      description: 'Retrieves all users who have signed up to the app',
+      parameters: [],
+      responses: {
+        200: response('Successfully retrieved profiles', {
+          message: {
+            type: 'string',
+            example: 'Successfully retrieved profiles',
+          },
+        }),
+        404: response("Ooops! There're no Users yet", {
+          error: { type: 'string', example: "Ooops! There're no Users yet" },
+        }),
+        500: response('Error occured while fetching profiles', {
+          error: {
+            type: 'string',
+            example: 'Error occured while fetching profiles',
+          },
+        }),
+      },
+    },
+  },
```

Comment:
How are you getting all users????
This doesn't make sense at all.
Your task is complete user profile